packages/fx-core/resource/package.nls.json

```diff
@@ -302,6 +306,40 @@
   "core.createProjectQuestion.capability.messageExtensionApiSpecOption.detail": "Create a message extension from your existing API",
   "core.createProjectQuestion.capability.copilotPluginAIPluginOption.label": "Start with an OpenAI Plugin",
   "core.createProjectQuestion.capability.copilotPluginAIPluginOption.detail": "Convert an OpenAI Plugin to Microsoft 365 Copilot plugin",
+  "core.createProjectQuestion.capability.customCopilotBasicOption.label": "Basic AI Chatbot",
+  "core.createProjectQuestion.capability.customCopilotBasicOption.detail": "Build a basic AI chatbot in Teams",
+  "core.createProjectQuestion.capability.customCopilotRagOption.label": "Chat With Your Data",
+  "core.createProjectQuestion.capability.customCopilotRagOption.detail": "Expand AI bot's knowledge with your content to get accurate answers to your questions",
+  "core.createProjectQuestion.capability.customCopilotAssistantOption.label": "AI Assistant",
+  "core.createProjectQuestion.capability.customCopilotAssistantOption.detail": "Build intelligent chatbot in Teams to complete everyday tasks efficiently",
+  "core.createProjectQuestion.capability.customCopilotRagCustomizeOption.label": "Customize",
+  "core.createProjectQuestion.capability.customCopilotRagCustomizeOption.detail": "Decide how to load your data",
+  "core.createProjectQuestion.capability.customCopilotRagAzureAISearchOption.label": "Azure AI Search",
+  "core.createProjectQuestion.capability.customCopilotRagAzureAISearchOption.detail": "Load your data from Azure AI Search service",
+  "core.createProjectQuestion.capability.customCopilotRagCustomApiOption.label": "Custom API",
+  "core.createProjectQuestion.capability.customCopilotRagCustomApiOption.detail": "Load your data from custom APIs based on OpenAPI description document",
+  "core.createProjectQuestion.capability.customCopilotRagMicrosoft365Option.label": "Microsoft 365",
+  "core.createProjectQuestion.capability.customCopilotRagMicrosoft365Option.detail": "Load your data from Microsoft Graph and SharePoint",
+  "core.createProjectQuestion.capability.customCopilotRag.title": "Chat With Your Data",
+  "core.createProjectQuestion.capability.customCopilotRag.placeholder": "Select an option to load your data",
+  "core.createProjectQuestion.capability.customCopilotAssistantNewOption.label": "Build New",
+  "core.createProjectQuestion.capability.customCopilotAssistantNewOption.detail": "Build your own AI Assistant from scratch using Teams AI Library",
+  "core.createProjectQuestion.capability.customCopilotAssistantAssistantsApiOption.label": "Build with Assistants API",
+  "core.createProjectQuestion.capability.customCopilotAssistantAssistantsApiOption.detail": "Build an intelligent AI Assistant with OpenAI Assistants API and Teams AI Library's built-in corrdination",
```

Comment:
Spelling error: Change 'corrdination' to 'coordination'.

Reply:
Ooops, thanks!

Reply:
Updated.